### ISSUE TITLE
Geotagging + Dtube earth map for regional video clustering

### ISSUE THREAD

**Issue by brishtiteveja** [dtube/dtube#334]
We can potentially ask users for location of upload. Youtube already does it. Snapchat and Zenly have similar things. Of course it will be optional!

The motivation is that users are interested in other users nearby. As this potentially helps them find friends or similar minded people or people in real world on the same platform.

If many users provide location, then we can have interesting heat map using some opensource maps
some examples/urls
https://github.com/mapbox
https://opensource.com/article/18/5/how-build-your-first-gis-app
github search with snapmap: https://github.com/search?q=snapmap
Use of mapbox: https://www.mapbox.com/showcase/snap

Also this will help with tagging.

Some screenshots borrowed from other apps like snapchat and zenl
![image0](https://user-images.githubusercontent.com/2281405/135275159-dc3cae26-4a3d-40a3-9c73-6b453fe1c176.png)
![image1](https://user-images.githubusercontent.com/2281405/135275165-7c92b008-3114-438e-9c17-523a417a59f9.png)
![image2](https://user-images.githubusercontent.com/2281405/135275175-149a0e1f-06e2-46e7-879c-76c6ef2fc7fc.png)
y.

May be also Bitmoji..
and Dtube bitmojis can be part of Dtube NFT #267

**Comment by tibfox** [comment]
Good idea- we could start by asking them for their location and store it as geolocation inside of the json metadata of posts. That's not a big deal but could lead us to have to include location permission on app store. And I can imagine that our current target userbase are not open to install an app that have this as required permission. We could make a poll or something about that in future.

Let's keep this in mind for now and work on it when we have a stable version
